images-list

```diff
@@ -535,12 +539,14 @@ quay.io/k8scsi/csi-attacher rancher/mirrored-k8scsi-csi-attacher v3.0.0
 quay.io/k8scsi/csi-attacher rancher/mirrored-k8scsi-csi-attacher v3.1.0
 quay.io/k8scsi/csi-node-driver-registrar rancher/mirrored-k8scsi-csi-node-driver-registrar v2.0.1
 quay.io/k8scsi/csi-node-driver-registrar rancher/mirrored-k8scsi-csi-node-driver-registrar v2.1.0
+quay.io/k8scsi/csi-node-driver-registrar rancher/mirrored-k8scsi-csi-node-driver-registrar v2.3.0
```

Comment:
Same as the  livenessprobe image, the registry changed. This should be:
`k8s.gcr.io/sig-storage/csi-node-driver-registrar rancher/mirrored-sig-storage-csi-node-driver-registrar v2.3.0`
New image: https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/28c55a22dcb986460b66f7de3838c96146e39900/manifests/vanilla/vsphere-csi-driver.yaml#L396

Reply:
Updated the PR with all the changes. it can be merged when EIO issue will be closed.